examples/htlc-pyteal-ts/README.md

```diff
@@ -1,4 +1,4 @@
-# Hash-Time-Lock-Contract Example using PyTeal
+# Hash-Time-Lock-Contract Example in Typescript using PyTeal
```

Comment:
can we add `Typescript Project` here ?

Reply:
What do you mean?

Reply:
something like this `Typescript Project - Hash-Time-Lock-Contract-Example using PyTeal`
or this one is ok too

Reply:
Ah, you suggest to update a title ;) 
Both are fine (the original, or the one you suggested)

Reply:
Keeping the original one :smile: